core/src/main/resources/hudson/model/View/AsynchPeople/index_pl.properties

```diff
@@ -25,3 +25,6 @@ Name=Nazwa
 On=Na
 People=U\u017Cytkownicy
 User\ Id=Identyfikator u\u017Cytkownika
+Last\ Commit\ Activity=Ostatnia zmiana
+blurb=Prezentuje wszystkich uwierzytelnionych u\u017Cytkownik\u00F3w w\u0142\u0105cznie z identyfikuj\u0105cym loginem oraz osoby wymienione w opisach zarejestrowanych zmianach.
```

Comment:
'identyfikującym loginem' -> rephrase to 'identyfikatorem'
'w opisach zarejestrowanych zmianach' -> 'w opisach zarejestrowanych zmian'